src/test/java/io/github/fbiville/nestedrelationquerysdn6/NestedRelationQuerySdn6ApplicationTests.java

```diff
@@ -14,65 +16,133 @@
 import org.testcontainers.junit.jupiter.Container;
 import org.testcontainers.junit.jupiter.Testcontainers;
 
+import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 @Testcontainers
 @DataNeo4jTest
+@TestInstance(Lifecycle.PER_CLASS)
 class NestedRelationQuerySdn6ApplicationTests {
 
     @Autowired
     private Driver driver;
 
-    @Container
-    private static final Neo4jContainer<?> neo4jContainer = new Neo4jContainer<>("neo4j:4.0");
+    //@Container
+    //private static final Neo4jContainer<?> neo4jContainer = new Neo4jContainer<>("neo4j:4.0");
```

Comment:
you do not have access to Docker at work? Is there an issue?

Reply:
Some hard VPN rules

Reply:
Is there an internal proxy you can point Docker to? If so, that could solve the issue.

Reply:
Probably, I didn't really try it that much, usually it's more about untrusted certificates.